pmlc/dialect/tile/transforms/BUILD

```diff
@@ -25,7 +25,7 @@ plaidml_cc_library(
 
 plaidml_cc_library(
     name = "transforms",
-    srcs = ["registration.cc"],
+    srcs = ["registration.cc", "constant_types.cc"],
```

Comment:
I think we should try to keep all registration in `registration.cc`

Reply:
Not sure how to do this, since `ConstantTypesPass` relies on `static llvm::cl::opt<std::string> constant_types_option_floatx`, which needs to be present in `constant_types.cc` for the default argument. Putting `constant_types_option_floatx` in a `consatnt_types.h` yields llvm errors due to multiple registration of the CL option.

Reply:
Can we declare the `llvm::cl::opt` in a header file and then have the `static` initializer (i.e. telling the compiler which translation unit this declaration should be defined in) live in `registration.cc`

Reply:
OK, this is fine the way it is, I can revisit this later perhaps. It's really an attempt to improve linker performance by minimizing the amount of `alwayslink=1` symbols to consider.